client/state/selectors/get-next-page-query.js

```diff
@@ -0,0 +1,27 @@
+/**
+ * Internal dependencies
+ */
+import getMediaQueryManager from 'state/selectors/get-media-query-manager';
+import getNextPageHandle from 'state/selectors/get-next-page-handle';
+
+const DEFAULT_QUERY = Object.freeze( { number: 20 } );
+
+/**
+ * Returns a new query object to use to fetch the next page of media for a site
+ *
+ * @param {object} state The state object
+ * @param {number} siteId The site ID
+ */
+export default function getNextPageQuery( state, siteId ) {
+	const queryManager = getMediaQueryManager( state, siteId );
+	if ( ! queryManager ) {
+		return DEFAULT_QUERY;
+	}
+
+	const currentQuery = queryManager.options.query;
```

Comment:
[Regarding my comment here](https://github.com/Automattic/wp-calypso/pull/44149#issuecomment-661812840) it seems like relevant query information is stored in the `queryRequests` reducer

<img width="432" alt="Screenshot 2020-07-21 at 14 10 24" src="https://user-images.githubusercontent.com/9202899/88053433-151eef00-cb5c-11ea-8920-a62ccf5a5758.png">

The `queryManager.options.query` returned `undefined` for me both on the initial source change and when filling in the a search field.

Reply:
I think `queryRequests` is the "current" request, not the full query object.

I'm really confused why `queryManager.options.query` isn't defined or why this isn't working as it was working when I first wrote it, but I wonder if I ran into an error and was blowing through compilation errors without noticing 😞

Based on https://github.com/Automattic/wp-calypso/blob/4ecbc4965a9cca5b90a09954d9a7276ea2d16726/client/state/media/reducer.js#L171 I think that `query` should be in the QueryManager, but maybe there's something wrong with this action 🤔

Reply:
should be fixed with 4b0682e